Frontend/src/app/components/top-bar/top-bar.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7378189)
```suggestion
import { Component, OnInit } from "@angular/core";
```